### PR TITLE
deps: regenerating with new Protobuf (6.23.x)

### PR DESCRIPTION
**Pull request by suztomo**
To merge into 6.23.x branch. This PR was generated using Protobuf 3.19.6. Source-Link: https://github.com/googleapis/googleapis/commit/6379d5fe706781af6682447f77f20d18b4db05b2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>${google.cloud.shared-dependencies.version}</version>
+        <version>2.10.1</version>
```
Comment (mpeddada1) [inline pom.xml:101]: We may need to manually update the shared-dependencies.version variable here?

**Comment by suztomo** [comment]
"Kokoro - Test: Integration" failed:

```
com.google.cloud.spanner.SpannerException: FAILED_PRECONDITION: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Invalid value for column json in table TestTable: Expected JSON.
	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerExceptionPreformatted(SpannerExceptionFactory.java:284)
	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerException(SpannerExceptionFactory.java:170)
	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerException(SpannerExceptionFactory.java:110)
	at com.google.cloud.spanner.TransactionRunnerImpl$TransactionContextImpl.commit(TransactionRunnerImpl.java:291)
	at com.google.cloud.spanner.TransactionRunnerImpl.lambda$runInternal$0(TransactionRunnerImpl.java:996)
	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:103)
	at com.google.cloud.RetryHelper.run(RetryHelper.java:76)

```

Retrying.

**Comment by rajatbhatta** [comment]
> "Kokoro - Test: Integration" failed:
> 
> ```
> com.google.cloud.spanner.SpannerException: FAILED_PRECONDITION: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Invalid value for column json in table TestTable: Expected JSON.
> 	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerExceptionPreformatted(SpannerExceptionFactory.java:284)
> 	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerException(SpannerExceptionFactory.java:170)
> 	at com.google.cloud.spanner.SpannerExceptionFactory.newSpannerException(SpannerExceptionFactory.java:110)
> 	at com.google.cloud.spanner.TransactionRunnerImpl$TransactionContextImpl.commit(TransactionRunnerImpl.java:291)
> 	at com.google.cloud.spanner.TransactionRunnerImpl.lambda$runInternal$0(TransactionRunnerImpl.java:996)
> 	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:103)
> 	at com.google.cloud.RetryHelper.run(RetryHelper.java:76)
> ```
> 
> Retrying.

Pushed a fix for this.

**Review by rajatbhatta — APPROVED** [review]

**Comment by rajatbhatta** [comment]
@suztomo: The Integration tests pass now. :)

**Comment by suztomo** [comment]
Thank you. I was about to ask you.

**Pull request merged**